src/python_testing/TC_OpstateCommon.py

```diff
@@ -369,7 +374,7 @@ async def TEST_TC_OPSTATE_BASE_2_1(self, endpoint=1):
 
         # STEP 4: TH reads from the DUT the CountdownTime attribute
         self.step(4)
-        if self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.A0002")):
+        if attributes.CountdownTime.attribute_id in attribute_list:
```

Comment:
One of the things that's getting removed here is that pics_guard automatically marks the step as skipped if the condition doesn't hold. This change removes that.

You had another PR were I commented that we should framework some of these attribute checks now that we have some functions in place to do it. https://github.com/project-chip/matter-test-scripts/issues/371. This might be a good PR to start this on, where the pics guard is directly swapped for something like attribute_guard (not totally sold on the naming, so if you can come up with a batter name, please do)

Reply:
Hi Cecille,
I have created a function in MatterBaseTest class named "attributes_guard" as you had mentioned using the check_attributes function in the matter_testing_scripts helper module.
This should work I hope, please see the update to the issue https://github.com/project-chip/matter-test-scripts/issues/370 you have filed and let me know if this will help resolve the above comment.
Thank you!